drivers/windows/dir_access_windows.cpp

```diff
@@ -286,7 +286,7 @@ Error DirAccessWindows::remove(String p_path) {
 
 	p_path = fix_path(p_path);
 
-	printf("erasing %s\n", p_path.utf8().get_data());
+	print_verbose("Erasing " + p_path.utf8().get_data());
```

Comment:
I think you can remove the print, `DirAccessUnix` doesn't print anything so it should be kept consistent. This was a debug print.

Reply:
Feel free to remove the dead code below too.

Reply:
Fixed :smiley: